Example/Tests/Classes/LPActionManagerTest.m

```diff
@@ -276,6 +277,7 @@ - (void)test_push_token
         return YES;
     }];
     [manager leanplum_application:app didRegisterForRemoteNotificationsWithDeviceToken:token];
+    [[LPOperationQueue serialQueue] waitUntilAllOperationsAreFinished];
```

Comment:
How long is the wait?

Reply:
This will wait until all added NSOperation are executed.